AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginFunctionalTests/README.md

```diff
@@ -34,7 +34,7 @@ enum PostStatus {
     PUBLISHED
 }
 
-type Post @model {
+type Post @model @auth(rules: [{ allow: public }]) {
```

Comment:
is this needed? I thought in V1 transformer, it is not deny by default and models can be accessed via APIKey if there are no `@auth` rules on it

Reply:
actually, I was getting warnings to use @auth rules, so when I updated the warnings stopped popping, no worrries - I've discarded the @auth changes.